dynomite/src/lib.rs

```diff
@@ -340,18 +340,19 @@
 #![deny(missing_docs)]
 // reexported
 // note: this is used inside the attr_map! macro
-#[cfg(feature = "default")]
-pub use rusoto_dynamodb_default as dynamodb;
-
-#[cfg(feature = "rustls")]
-pub use rusoto_dynamodb_rustls as dynamodb;
```

Comment:
Would it be possible to still reexport this under dynamodb? We do this as a convenience to users who would otherwise need to declare two dependencies to use this crate.

Reply:
I think I did down below (line 355). Is that not what you mean?

Reply:
Sorry for the confusion, I should have placed it right next to the lines I removed to be clear